src/go/k8s/tests/e2e/console-admin-api/03-call-console.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: call-console
+  namespace: console-admin-api
+spec:
+  template:
+    spec:
+      containers:
+        - name: curl
+          image: curlimages/curl:latest
+          env:
+            - name: POD_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+          command:
+            - /bin/sh
+            - -c
+            - -ex
+          args:
+            - curl http://console.$POD_NAMESPACE.svc.cluster.local:8080/admin/health -v
```

Comment:
What this test verifies? Am I right in thinking any none 2XX HTTP response from console will fail this job?

Reply:
Console has a lot of checks before it can start the http server, this check just verifies that the endpoint is up but not the http status. I don't know if there is an easy way to do that other than complicated bash :thinking:

Reply:
@nicolaferraro already provide examples of this in:
https://github.com/redpanda-data/redpanda/blob/909dc2623f71e1233b9b21a5bd8dfdbe75563bd2/src/go/k8s/tests/e2e/explicit-ports-tls/01-probes.yaml#L24-L29
https://github.com/redpanda-data/redpanda/blob/2738403e6c3c60e3708d3ea1dc4abed59f70e202/src/go/k8s/tests/e2e/decommission/04-probe.yaml#L23-L29
https://github.com/redpanda-data/redpanda/blob/2738403e6c3c60e3708d3ea1dc4abed59f70e202/src/go/k8s/tests/e2e/decommission/04-assert.yaml#L7-L13
and more

Reply:
Updated